client/signup/site-mockup/index.jsx

```diff
@@ -43,6 +44,31 @@ class SiteMockups extends Component {
 		verticalData: {},
 	};
 
+	constructor( props ) {
+		super( props );
+		this.state = this.getFontLoaderState( props );
+	}
+
+	getFontLoaderState( props ) {
+		const state = {
+			fontLoaded: false,
+			fontLoader: loadFont( props.siteStyle, props.siteType ),
```

Comment:
Is `this.state.fontLoader()` used anywhere? Not a biggie, but could we just `loadFont` straight up without assignment? 

```
		loadFont( props.siteStyle, props.siteType )
			.then( () => this.setState( { fontLoaded: true } ) );
```

Reply:
Yeah, it's a local state but not in any way tied to rendering so it shouldn't really be on `this.state`. Good catch.